packages/web-components/src/radio/README.md

```diff
@@ -0,0 +1,122 @@
+# Radio
```

Comment:
We should couple RadioGroup along with this I think. If the goal is to leverage the FAST implementation, those two go hand in hand since we implement non-native inputs. All the behavior is in RadioGroup as of now.